src/rules/no-try-expect.ts

```diff
@@ -13,13 +16,18 @@ export default {
         'or "await expect(yourFunction()).rejects.toThrow()" for async tests',
       ].join(' '),
     },
+    type: 'problem',
+    schema: [],
   },
+  defaultOptions: [],
   create(context) {
     let isTest = false;
     let catchDepth = 0;
 
-    function isThrowExpectCall(node) {
-      return catchDepth > 0 && node.callee.name === 'expect';
+    function isThrowExpectCall(node: TSESTree.CallExpression) {
+      return (
+        catchDepth > 0 && 'name' in node.callee && node.callee.name === 'expect'
```

Comment:
Can we use `isExpectCall`?

Reply:
we can 😀

Reply:
I'll be sure to make sure that doesn't change when cleaning up the new guards